.github/workflows/pr-e2e.yml

```diff
@@ -155,6 +155,7 @@ jobs:
           AZURE_RUN_AAD_POD_IDENTITY_TESTS: true
           AZURE_RUN_WORKLOAD_IDENTITY_TESTS: true
           GCP_RUN_IDENTITY_TESTS: true
+          ENABLE_OPENTELEMETRY : true
```

Comment:
As this file is got from `main` branch, we can't test this right now. We have 2 options (and both are valid at least to me), 
- Adding this field (here an also in the other yaml) in other PR that we can merge before this
- Removing this field from here and the other yaml and I can add it as secret in the repo (which is dynamically loaded so it doesn't require any other PR)

I guess that to be consistent with other flags, we should add it in other split PR, but as you prefer